docker/pravega/scripts/lts_recovery_backup.sh

```diff
@@ -80,6 +80,8 @@ do
   retry=$((retry + 1))
 done
 
+sleep 120s
```

Comment:
only on success you can sleep this while. i.e before L87

Reply:
Made the changes.